src/mongocxx/test/uri.cpp

```diff
@@ -67,7 +67,7 @@ TEST_CASE("URI", "[uri]") {
         REQUIRE(u.hosts().size() == 1);        
         REQUIRE(u.hosts()[0].name == "localhost");
         REQUIRE(u.hosts()[0].port == 27017);
-        REQUIRE(u.hosts()[0].family == 0);
```

Comment:
There's presumably a #define'd constant for this in socket.h on Linux/Mac (AF_INET?), but it's elsewhere on Windows...